godot-core/src/registry/signal/connect_builder.rs

```diff
@@ -125,13 +127,14 @@ where
     fn inner_connect_godot_fn<F>(
         self,
         godot_fn: impl FnMut(&[&Variant]) -> Result<Variant, ()> + 'static,
+        bound: &Gd<impl GodotClass>,
```

Comment:
Note: I'm not using an Option on purpose – Signal Callables **must** be bound to some object, either emitter or receiver. Otherwise they are not properly displayed&handled by Godot Runtime